libmscore/sym.h

```diff
@@ -2913,7 +2913,7 @@ class Sym {
    public:
       Sym() { }
 
-      bool isValid() const                       { return _code != -1; }
+      bool isValid() const                       { return _code != -1 && _bbox.isValid(); }
```

Comment:
This check seems to have broken the ability to add the space character from the Special Characters dialog.  Not sure if this check was necessary, or if there is perhaps a way to give space a valid bbox, or if we could just exempt space from the check